### PR TITLE
Update set-up-conferencing-policies-for-your-organization.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-policies-in-your-organization/set-up-conferencing-policies-for-your-organization.md [hunk 1]
```diff
@@ -35,41 +35,24 @@ Policy settings can be configured at the time a policy is created, or you can us
...
-	 > If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
+### Start Windows PowerShell
 
-	   ```PowerShell      
-		Import-Module -Name MicrosoftTeams
-		$credential = Get-Credential
-		$session = New-CsOnlineSession -Credential $credential
-		Import-PSSession $session
-	   ```
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/set-up-conferencing-policies-for-your-organization.md:41]:
```suggestion
> [!NOTE]
```

File: Skype/SfbOnline/set-up-policies-in-your-organization/set-up-conferencing-policies-for-your-organization.md [hunk 2]
```diff
@@ -35,41 +35,24 @@ Policy settings can be configured at the time a policy is created, or you can us
...
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-policies-in-your-organization/set-up-conferencing-policies-for-your-organization.md:42]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**